src/toolkits/activity_classification/ac_data_iterator.hpp

```diff
@@ -61,8 +61,11 @@ class data_iterator {
      */
     flex_list class_labels;
 
-    /**  Generates verbose output when set to true. */
-    bool verbose;
+    /**  Set to true, when the data is used for training. */
+    bool is_train;
+
+    /** Augments training data when set to true*/
+    bool use_data_augmentation;
```

Comment:
For data members with primitive types, such as int, bool, and float, it’s often a good idea to give them a default value (such as for `predictions_in_chunk` above). This way, forgetting to set them later will yield a deterministic result, instead of getting some non deterministic, uninitialized value.